src/window/editor/scenario_event_details.c

```diff
@@ -121,14 +121,14 @@ static struct {
 } data;
 
 static input_box event_name_input = {
-    112, 50, 32, 2, FONT_NORMAL_WHITE, 1, data.event_name, EVENT_NAME_LENGTH
+    100, 40, 32, 2, FONT_NORMAL_WHITE, 1, data.event_name, EVENT_NAME_LENGTH
 };
 
 static grid_box_type conditions_grid_box = {
     .x = 16,
-    .y = 228,
+    .y = 188,
     .width = 18 * BLOCK_SIZE,
-    .height = 10 * BLOCK_SIZE,
+    .height = 10 * BLOCK_SIZE + 52,
```

Comment:
```suggestion
    .height = 13 * BLOCK_SIZE,
```

Not exactly the same height and not sure it would fit 7 items, but it would be aligned to 16 pixels.

Reply:
10*16+52=212px (Exactly under 7 lines )
13*16=208px  (7 line won't fit)

Reply:
Fair enough. Then:

```suggestion
    .height = 13 * BLOCK_SIZE + 4,
```

Same thing below.